test-resources/msgpack_test_fixtures.json

```diff
@@ -0,0 +1,75 @@
+[
+  {
+    "Name": "json array",
```

Comment:
The other JSON files in this directory use camelCase for the keys, should this file conform to that convention?

Reply:
you mean snake_case

Yes, fair point. I'll fix it

Reply:
I did mean camelCase (e.g. [`expectedType`](https://github.com/ably/ably-common/blob/main/test-resources/messages-encoding.json#L7) or [`pushEnabled`](https://github.com/ably/ably-common/blob/main/test-resources/test-app-setup.json#L22), as opposed to PascalCase `ExpectedType` or `PushEnabled`)

Reply:
done